models/dex/chains/mev/arbitrages/dex_arbitrum_atomic_arbitrages.sql

```diff
@@ -0,0 +1,15 @@
+{{ config(
+        schema = 'dex_arbitrum',
+        alias = 'atomic_arbitrages',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['blockchain','tx_hash', 'evt_index']
```

Comment:
can we look to add that target incremental filter on these too, similar to the dex sandwiched?